x-pack/filebeat/module/ibmmq/_meta/docs.asciidoc

```diff
@@ -0,0 +1,43 @@
+:modulename: ibmmq
+
+== ibmmq module
+
+This is the ibmmq module.
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+TODO: document with what versions of the software is this tested
```

Comment:
@felix-lessoer Can you provide these details here?

Reply:
I tested in v 9.1.0.0 .. But the logs are the same in older versions

Reply:
I think we can just write that, it's good to explain what versions have we tested against, even if it's expected to work with others